src/cli/commands/codegen.ts

```diff
@@ -17,12 +17,13 @@ export default {
     try {
       await new CodeGenerator(config.get('GENERATED_FOLDER'), config.get('DB_ENTITIES'), {
         resolversPath: config.get('RESOLVERS_PATH'),
+        validateResolvers: config.get('VALIDATE_RESOLVERS') === 'true',
         warthogImportPath: config.get('MODULE_IMPORT_PATH')
       }).generate();
     } catch (error) {
       console.error(error); // eslint-disable-line
     }
 
-    cleanUpTestData();
+    await cleanUpTestData();
```

Comment:
I changed this line because it failed precommit linting, which was triggered by my other change in this file.

Scenarios like this could be avoided in the future if Circle ran linting on the entire repo as part of the build. That way, you never have surprise linting errors that can be merged in (usually through `git commit --no-verify`) and pop back up when someone makes an unrelated change in the affected file at a later point.

Reply:
Good call, will add a git issue for this. 👍